src/OpenLoco/src/S5/S5.h

```diff
@@ -7,6 +7,7 @@
 #include "ScenarioObjective.h"
 #include <cstdint>
 #include <memory>
+#include <optional>
```

Comment:
Is this used? I only see `std::unique_ptr` as a new type in the diff.

Reply:
no, good point.